backend/organization/models/organization.py

```diff
@@ -24,8 +24,6 @@ class Organization(models.Model):
         help_text="Points to organization url slug",
         verbose_name="URL Slug",
         max_length=1024,
-        null=True,
-        blank=True,
         unique=True
```

Comment:
So the problem here removing null=True and blank=True is. You are required to insert text to this column at the time of row creation. Currently we insert name + ID as a url_slug. So I think we should keep this as it was before. Otherwise there will conflict when you create a row.

Reply:
makes sense!